### ISSUE TITLE
GridLayoutManager topDivider 

### ISSUE THREAD

**Issue by stormlei** [Fondesa/RecyclerViewDivider#21]
How does the top line show up? @Fondesa

**Comment by stormlei** [comment]
thanks

**Comment by fondesa** [comment]
@stormlei  Since other users asked it, I'll add this feature in the milestone [4.x](https://github.com/fondesa/recycler-view-divider/milestone/2).
The updates will be done on the issue #68.

**Comment by fondesa** [comment]
Just released the version [3.2.0](https://github.com/fondesa/recycler-view-divider/releases/tag/3.2.0) which contains `showFirstDivider` to show the top divider.

**e.g.** (kotlin)

```kotlin
context.dividerBuilder()
    .showFirstDivider()
    .build()
    .addTo(recyclerView)
```
